lib/Elastica/Index/Settings.php

```
@@ -68,31 +68,33 @@ public function get($setting = '')
         $requestData = $this->request()->getData();
         $data = reset($requestData);
 
+        if (empty($data['settings']) || empty($data['settings']['index'])) {
+            return []; // index not found
```

Should we actually throw a not found exception here?